concepticondata/conceptlists/Steinthal-1875-1565.tsv

```diff
@@ -0,0 +1,1566 @@
+Steinthal-1875-1565-686	686	sehen	sehen	to see	1409	SEE
+Steinthal-1875-1565-687	687	erblicken	erblicken	to catch sight of something or someone		
+Steinthal-1875-1565-688	688	finden	finden	to find	719	FIND
+Steinthal-1875-1565-689	689	spähen	spähen	to scout		
+Steinthal-1875-1565-690	690	hören	hören	to hear	1408	HEAR
+Steinthal-1875-1565-691	691	lauschen	lauschen	to listen	1608	LISTEN
+Steinthal-1875-1565-692	692	riechen	riechen	to smell	1586	SMELL (PERCEIVE)
+Steinthal-1875-1565-693	693	schmecken	schmecken	to taste	21	TASTE (SOMETHING)
+Steinthal-1875-1565-694	694	tasten	tasten	to touch	2559	FEEL (TACTUALLY)
+Steinthal-1875-1565-695	695	fühlen	fühlen	to feel	1079	FEEL
+Steinthal-1875-1565-696	696	schlafen	schlafen	to sleep	1585	SLEEP
+Steinthal-1875-1565-697	697	schläfrig sein	schläfrig sein	to be tired	1757	TIRED
+Steinthal-1875-1565-698	698	schnarchen	schnarchen	to snore	1983	SNORE
+Steinthal-1875-1565-699	699	träumen	träumen	to dream	1920	DREAM (SOMETHING)
+Steinthal-1875-1565-700	700	sich niederlegen	sich niederlegen	to lie down	215	LIE DOWN
+Steinthal-1875-1565-701	701	aufwachen	aufwachen	to wake up	1602	WAKE UP
+Steinthal-1875-1565-702	702	aufstehn	aufstehn	to get up	1512	RISE (ONESELF)
+Steinthal-1875-1565-703	703	wachen bei einem Kranken	wachen bei einem Kranken	to keep watch over a sick person		
+Steinthal-1875-1565-704	704	Ich bin wach.	Ich bin wach.	I am awake.	825	AWAKE
+Steinthal-1875-1565-705	705	wohnen	wohnen	to reside	363	DWELL (LIVE, RESIDE)
+Steinthal-1875-1565-706	706	leben	leben	to live	2309	LIVE
+Steinthal-1875-1565-707	707	wachsen	wachsen	to grow	479	GROW
+Steinthal-1875-1565-708	708	sterben	sterben	to die	1494	DIE
+Steinthal-1875-1565-709	709	tödten	tödten	to kill	1417	KILL
+Steinthal-1875-1565-710	710	gesund	gesund	healthy	1364	HEALTHY
+Steinthal-1875-1565-711	711	kräftig	kräftig	strong	785	STRONG
+Steinthal-1875-1565-712	712	schwach	schwach	weak	1601	WEAK
+Steinthal-1875-1565-713	713	krank	krank	sick	1847	SICK
+Steinthal-1875-1565-714	714	gross	gross	tall, big	1202	BIG
+Steinthal-1875-1565-715	715	klein	klein	small	1246	SMALL
+Steinthal-1875-1565-716	716	nackt	nackt	naked	590	NAKED
+Steinthal-1875-1565-717	717	Schmerz, Kopf-, Leib-	Schmerz	pain	1783	PAIN
+Steinthal-1875-1565-718	718	Schmerz, Kopf-, Leib-	Kopfschmerz	headache	3625	HEADACHE
+Steinthal-1875-1565-719	719	Schmerz, Kopf-, Leib-	Leibschmerz	bellyache		
+Steinthal-1875-1565-720	720	Fieber	Fieber	fever	1534	FEVER
+Steinthal-1875-1565-721	721	Ausschlag	Ausschlag	rash		
+Steinthal-1875-1565-722	722	Blattern	Blattern	smallpox	1054	SMALLPOX
+Steinthal-1875-1565-723	723	sonstige heimische Krankheiten	sonstige heimische Krankheiten	other local diseases		
+Steinthal-1875-1565-724	724	Die Wunde schmerzt, heilt.	Die Wunde schmerzt.	The wound hurts.		
+Steinthal-1875-1565-725	725	Die Wunde schmerzt, heilt.	Die Wunde heilt.	The wound heals.		
+Steinthal-1875-1565-726	726	Es juckt mich.	Es juckt mich.	I am itching.		
```

Comment:
149 ITCHY?

Reply:
Here I was again hesitant because of the sentence. But since you both agree, I will map it to that.